submissions/YuraZagor/tjsw/index.js

```diff
@@ -0,0 +1,154 @@
+const manNamesArr = ['Vasil', 'Yurij', 'Zahar', 'Taras', 'Olexiy', 'Oleg', 'Mykola', 'Sashko', 'Sergij', 'Bohdan']
+const womanNamesArr = ['Hanna', 'Helena', 'Vasilisa', 'Yulya', 'Oksana', 'Olena', 'Olga', 'Myroslava', 'Svitlana', 'Bohdana']
+const animalNamesArr = ['Joy', 'Happy', 'Lucky', 'Puffy']
+
+let catSay = 'meau';
+let dogSay = 'woufff';
+let humanSay = 'Glory to Ukraine!';
+
+function randomSelect(arr){
+  return arr[Math.floor(Math.random()*arr.length)]
+}
+let manName = randomSelect(manNamesArr)
+let womanName = randomSelect(womanNamesArr)
+let animalName = randomSelect(animalNamesArr)
+
+const inhabitants = [
+  { species: 'human', 
+    gender: 'male',
+    name: 'Vasil',
+    saying: 'Slava natsiyi',
+    friend: 'Hanna',
+    legs: 2,
+    hands: 2,
+  },
+  { species: 'human', 
+    gender: 'female',
+    name: 'Hanna',
+    saying: 'Smert vorogam',
+    friend: 'Vasil',
+    legs: 2,
+    hands: 2,
+  },
+];
+
+class Inhabitant {  
+  constructor(species, name, saying, friend, gender, habitat=inhabitants){
+    this.species=species;
+    this.name=name;
+    this.saying = saying
+    this.friend=friend;
+    this.gender=gender;
+    this.habitat = habitat
```

Comment:
For the love of God make your code style consistent.
Looks like Frakestein-style composition of code from various sources.